x/ibc/07-tendermint/types/misbehaviour_handle.go

```diff
@@ -25,29 +25,29 @@ func (cs ClientState) CheckMisbehaviourAndUpdateState(
 	clientStore sdk.KVStore,
 	misbehaviour clientexported.Misbehaviour,
 ) (clientexported.ClientState, error) {
+	tmEvidence, ok := misbehaviour.(*Misbehaviour)
```

Comment:
```suggestion
	tmMisbehaviour, ok := misbehaviour.(*Misbehaviour)
```
would need to change other refs as well